keps/sig-apps/3939-consider-terminating-pods-jobs/README.md

```diff
@@ -201,11 +201,14 @@ before starting new pods.
 
 In scarce compute environments, these resources can be difficult to obtain so pods can take a long time to find resources and they may only be able to find nodes once the existing pods have been terminated.
 
+If a job is stuck in terminating, it could be possible for autoscaling to kick in and give a new node.  
+This is not ideal if you could just wait for the pod to terminating and reuse that node.  
+
 ### Goals
 
-- Deployment controller should allow for flexibility in waiting for pods to be fully terminated before
+- Job controller should allow for flexibility in waiting for pods to be fully terminated before
   creating new ones
-- Deployment/ReplicaSet will have a new status field where we include the number of terminating replicas.
+- Job controller will have a new status field where we include the number of terminating replicas.
 
 ### Non-Goals
 
```

Comment:
You can say, "Other workload APIs are not included in this proposal", to account for Deployment/ReplicaSet, now removed.

Reply:
done.